recipes/nasm/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+import os
+from conans import ConanFile, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch",
```

Comment:
```suggestion
    settings = "os", "arch", "build_type", "compiler"
```
Always keep full settings.

Reply:
ok, it was like that before the upgrade but i'll change it